src/main/java/com/pkb/unit/dot/DOT.java

```diff
@@ -8,18 +8,31 @@
 import com.pkb.unit.tracker.Unit;
 
 /**
- * Class for transforming the current state of the registry into a DOT format graph
+ * Class for transforming the current state of the system into a DOT format graph
  * https://graphviz.gitlab.io/_pages/doc/info/lang.html
  */
 public class DOT {
     private DOT() {}
 
+    /**
+     * Converts the given {@link SystemState} into a DOT format graph.
+     * @see <a href="https://graphviz.gitlab.io/_pages/doc/info/lang.html" a>DOT language</a>
```

Comment:
Is there an extra "a"?
```suggestion
     * @see <a href="https://graphviz.gitlab.io/_pages/doc/info/lang.html">DOT language</a>
```

Reply:
Wow, great spot!